domain/src/main/java/org/akvo/flow/domain/interactor/datapoints/DownloadMedia.kt

```diff
@@ -64,6 +64,6 @@ class DownloadMedia @Inject constructor(
     }
 
     companion object {
-        const val PARAM_FILE_PATH = "data_point_id"
+        const val PARAM_FILE_PATH = "file_path"
```

Comment:
Yep this is out of scope but stumbled upon it and just considered an overkill to create a PR just for that one small change.